### PR TITLE
Extend gravity report docs

### PR DESCRIPTION
**Pull request by bernardjkim**
## Description

This PR updates the `gravity report` docs. Adds additional information on what information is collected and contained in the report tarball. Also adds some examples on using the lower level `gravity system report` command.

## Type of change

* Documentation

## Linked tickets and other PRs

* Refs https://github.com/gravitational/gravity/issues/2102

## TODOs

- [x] Address review feedback

### CODE REVIEW

**Comment by bernardjkim** [comment]
@r0mant 
The gravity documentation already contains a section on using the `gravity report` command. I've added a bit more information on what logs and information is collected in the tarball.

Are there any specific example use cases that you think would be helpful to add to the docs?

File: docs/7.x/cluster.md [hunk 1]
```diff
@@ -1647,4 +1647,43 @@ Example:
   $ gravity report --since 0s
 ```
 
-This command will collect diagnostics from all Cluster nodes into the specified tarball that you can then submit for evaluation.
+This command will collect diagnostics from all Cluster nodes into the specified tarball that you can then submit for
+evaluation.
+
+If you would like to examine the debug report, here is a quick overview of what information is collected and availble
```
Comment (a-palchikov) [inline docs/7.x/cluster.md:1653]: avail**a**ble

File: docs/7.x/cluster.md [hunk 2]
```diff
@@ -1647,4 +1647,43 @@ Example:
...
+for examination. First extract the tarball into a directory of your choice. Here we'll extract the tarball into a
+`report` directory with: `mkdir report && tar -zxf report.tar.gz -C report`.
+
+The contents of the report should look something like:
```
Comment (a-palchikov) [inline docs/7.x/cluster.md:1657]: nit: similar to this:

File: docs/7.x/cluster.md [hunk 3]
```diff
@@ -1647,4 +1647,43 @@ Example:
...
+├── node-2-etcd.tar.gz
+├── node-3-debug-logs.tar.gz
+├── node-3-etcd.tar.gz
+├── node-3-k8s-logs.tar.gz
+├── node-3-resources.tar.gz
+├── node-3-status.tar.gz
+└── operation_install.a27a9fa0-c5e6-4a62-973b-2893752c25b6
+```
+
+* `cluster.json` contains the JSON-encoded cluster metadata. Pretty print the contents using the python json tool:
```
Comment (a-palchikov) [inline docs/7.x/cluster.md:1674]: hmm, we should probably be doing this automatically then

File: docs/7.x/cluster.md [hunk 4]
```diff
@@ -1647,4 +1647,42 @@ Example:
...
+├── node-2-etcd.tar.gz
+├── node-3-debug-logs.tar.gz
+├── node-3-etcd.tar.gz
+├── node-3-k8s-logs.tar.gz
+├── node-3-resources.tar.gz
+├── node-3-status.tar.gz
+└── operation_install.a27a9fa0-c5e6-4a62-973b-2893752c25b6
+```
+
+* `cluster.json` contains the JSON-encoded cluster metadata.
```
Comment (r0mant) [inline docs/7.x/cluster.md:1674]: Nit: I wonder if structuring this information as a table would be a bit cleaner.

**Review by r0mant — COMMENTED** [review]
Can we also add information about collecting report from individual nodes using lower level `gravity system report` command?

File: tool/gravity/cli/register.go [hunk 5]
```diff
@@ -723,7 +723,7 @@ func RegisterCommands(app *kingpin.Application) *Application {
 	g.SystemServiceJournalCmd.Args = g.SystemServiceJournalCmd.Arg("arg", "optional arguments to the journalctl").Strings()
 
 	g.SystemReportCmd.CmdClause = g.SystemCmd.Command("report", "collect system diagnostics and output as gzipped tarball to terminal").Hidden()
-	g.SystemReportCmd.Filter = g.SystemReportCmd.Flag("filter", "collect only specific diagnostics ('system', 'kubernetes'). Collect everything if unspecified").Strings()
+	g.SystemReportCmd.Filter = g.SystemReportCmd.Flag("filter", "collect only specific diagnostics ('system', 'kubernetes', 'etc', 'timeline', 'resources'). Collect everything if unspecified").Strings()
```
Comment (a-palchikov) [inline tool/gravity/cli/register.go:726]: nit: we should list these in a variable that is updated whenever a new report filter type is added.

**Review by a-palchikov — APPROVED** [review]

**Pull request merged**